nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# creating a payment type will now validate expiration date

Fixes:
-
-

Changes proposed in this request & reasons behind organizational or architectural decisions:
- PaymentType model now has ExpirationDate property
-

Steps to test:
- open migration console with Ctrl + .
- run update-database command (this should add the ExpirationDate column to the PaymentType table)
- go to Profile, create a new payment type
- if you set the expiration date before today, it should notify you with an error

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-
-

Commit message:
-

Link to feature ticket:
-

@mighty-mahi-mahi
